rosdep/python.yaml

```diff
@@ -1649,6 +1650,7 @@ python-future:
   ubuntu:
     artful: [python-future]
     bionic: [python-future]
+    focal: [python3-future]
```

Comment:
Again, this is a python 3 package on a Python 2 key, and again, I don't think there exists a Python 2 package to satisfy this dependency.